.github/ ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
This is to hide the highlighted link in the screenshot. Users can still create an issue with a different format by clearing out all the text from the one of the existing templates.
![image](https://user-images.githubusercontent.com/1319033/94230608-f81de800-ff1f-11ea-997a-5bc4c41c0148.png)